zebrad/src/commands/config.rs

```diff
@@ -27,7 +27,7 @@ impl Runnable for ConfigCmd {
 # can be found in Rustdoc. XXX add link to rendered docs.
 
 "
-        .to_owned();
+        .to_owned(); // The default name and location of the config file is defined in ../commands.rs
```

Comment:
:+1: subsequent to and separately from this PR we could probably add a note to the help text.

Reply:
https://github.com/ZcashFoundation/zebra/issues/95